mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,28 +1,28 @@
 CLASS agw net/minecraft/inventory/Inventory
-	METHOD K_ getInvMaxStackAmount ()I
-	METHOD N_ getInvSize ()I
-	METHOD a getInvStack (I)Lbcb;
+	METHOD K_ getSlotCapacity ()I
+	METHOD N_ getInventorySize ()I
+	METHOD a getStack (I)Lbcb;
 		ARG 1 slot
-	METHOD a takeInvStack (II)Lbcb;
+	METHOD a removeStack (II)Lbcb;
 		ARG 1 slot
 		ARG 2 amount
-	METHOD a setInvStack (ILbcb;)V
+	METHOD a setStack (ILbcb;)V
 		ARG 1 slot
 		ARG 2 stack
-	METHOD a canPlayerUseInv (Lavy;)Z
+	METHOD a canAccessInventory (Lavy;)Z
 		ARG 1 player
-	METHOD a getInvAmountOf (Lbbw;)I
+	METHOD a getTotalStackCount (Lbbw;)I
 		ARG 1 item
-	METHOD a containsAnyInInv (Ljava/util/Set;)Z
+	METHOD a anyMatch (Ljava/util/Set;)Z
 		ARG 1 items
-	METHOD a_ onInvOpen (Lavy;)V
+	METHOD a_ onInventoryOpened (Lavy;)V
 		ARG 1 player
-	METHOD b removeInvStack (I)Lbcb;
+	METHOD b removeStack (I)Lbcb;
 		ARG 1 slot
-	METHOD b isValidInvStack (ILbcb;)Z
+	METHOD b isValidStack (ILbcb;)Z
 		ARG 1 slot
 		ARG 2 stack
-	METHOD b_ onInvClose (Lavy;)V
+	METHOD b_ onInventoryClosed (Lavy;)V
 		ARG 1 player
-	METHOD c isInvEmpty ()Z
+	METHOD c hasStacks ()Z
```

Comment:
Maybe I'm going crazy, but isn't this opposite (function returns true if it's empty)? I think isEmpty may be a better choice.

Reply:
My god... Don't map code at 3am, kids. I will fix this tomorrow. Fml.